request/graphql/schema/descriptions.go

```diff
@@ -23,14 +24,15 @@ var (
 
 	//nolint:unused
 	gqlTypeToFieldKindReference = map[gql.Type]client.FieldKind{
-		gql.ID:        client.FieldKind_DocKey,
-		gql.Boolean:   client.FieldKind_BOOL,
-		gql.Int:       client.FieldKind_INT,
-		gql.Float:     client.FieldKind_FLOAT,
-		gql.DateTime:  client.FieldKind_DATETIME,
-		gql.String:    client.FieldKind_STRING,
-		&gql.Object{}: client.FieldKind_FOREIGN_OBJECT,
-		&gql.List{}:   client.FieldKind_FOREIGN_OBJECT_ARRAY,
+		gql.ID:                      client.FieldKind_DocKey,
+		gql.Boolean:                 client.FieldKind_BOOL,
+		gql.Int:                     client.FieldKind_INT,
+		gql.Float:                   client.FieldKind_FLOAT,
+		gql.DateTime:                client.FieldKind_DATETIME,
+		gql.String:                  client.FieldKind_STRING,
+		&gql.Object{}:               client.FieldKind_FOREIGN_OBJECT,
+		&gql.List{}:                 client.FieldKind_FOREIGN_OBJECT_ARRAY,
+		schemaTypes.BytesScalarType: client.FieldKind_BYTES,
```

Comment:
suggestion: Related to the other comment from Fred, I think it should be made clear that these are custom scalars, as the comment on line 36 suggests ("More custom ones to come"). Just a quick comment separating base scalars from custom scalars for clarity 👍